_words/writing-service-pages.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Writing Service Pages
+position: 10
+---
+
+{::options auto_ids="false"/}
+
+Quarter hulk pinnace dance the hempen jig Chain Shot haul wind cable hearties hail-shot no prey, no pay. Clap of thunder Buccaneer scourge of the seven seas ahoy swing the lead Sea Legs cable come about fore American Main. Dance the hempen jig fluke scourge of the seven seas hang the jib scuppers execution dock Blimey jolly boat bucko scurvy.
+
+Salmagundi Jack Tar driver Admiral of the Black gally scuppers sheet lass Sea Legs gabion. Lookout killick hulk aft ahoy maroon Admiral of the Black aye poop deck overhaul. Come about keelhaul long clothes driver man-of-war careen sutler Davy Jones' Locker ye pressgang.
+
+<div></div>
+{: #title .anchor}
```

Comment:
If anyone has an idea for a better way to do this I'd love to not have little <div>s sitting around. Maybe custom tags? I'm gonna do a bit more jekyll research.